app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -79,3 +82,7 @@
     margin: $cads-spacing-6 0;
   }
 }
+
+.js-supplier-table-count-text {
+  position: absolute;
```

Comment:
Same comment about styling a `.js-*` selector applies to this - but if you rearrange the haml you can just get rid of this

Reply:
now removed